src/components/NotificationsList.js

```diff
@@ -22,24 +22,27 @@ class NotificationsList extends Component {
 
   componentDidMount() {
     if (this.props.navigation.state.routeName === 'commerceNotificationslist') {
-      this.props.onCommerceNotificationsRead(this.props.commerceId);
+      this.unsubscribeNotificationsRead = this.props.onCommerceNotificationsRead(this.props.commerceId);
       this.setState({ type: 'commerce' });
     } else {
-      this.props.onClientNotificationsRead(this.props.clientId);
+      this.unsubscribeNotificationsRead = this.props.onClientNotificationsRead(this.props.clientId);
       this.setState({ type: 'client' });
     }
   }
 
```

Comment:
`onNotificationsRead = () => {
    this.unsubscribeNotificationsRead && this.unsubscribeNotificationsRead();

    if (this.props.navigation.state.routeName === 'commerceNotificationslist') {
      this.unsubscribeNotificationsRead = this.props.onCommerceNotificationsRead(this.props.commerceId);
      this.setState({ type: 'commerce' });
    } else {
      this.unsubscribeNotificationsRead = this.props.onClientNotificationsRead(this.props.clientId);
      this.setState({ type: 'client' });
    }
  }`

definis esa funcion, y la tenes que llamar en el componentDidMount() y en el onRefresh del refresh control, en la prop refreshing tenes que dejar como estaba antes